packages/state-transition/src/epoch/processSlashings.ts

```diff
@@ -50,15 +50,23 @@ export function processSlashings(
     totalBalanceByIncrement
   );
   const increment = EFFECTIVE_BALANCE_INCREMENT;
+
+  const penaltyPerEffectiveBalanceIncrement = Math.floor(
+    (adjustedTotalSlashingBalanceByIncrement * increment) / totalBalanceByIncrement
+  );
   const penalties: number[] = [];
 
   const penaltiesByEffectiveBalanceIncrement = new Map<number, number>();
   for (const index of cache.indicesToSlash) {
     const effectiveBalanceIncrement = effectiveBalanceIncrements[index];
     let penalty = penaltiesByEffectiveBalanceIncrement.get(effectiveBalanceIncrement);
     if (penalty === undefined) {
-      const penaltyNumeratorByIncrement = effectiveBalanceIncrement * adjustedTotalSlashingBalanceByIncrement;
-      penalty = Math.floor(penaltyNumeratorByIncrement / totalBalanceByIncrement) * increment;
+      if (fork < ForkSeq.electra) {
```

Comment:
any reason why the check is reversed here?

Reply:
> any reason why the check is reversed here?

No particular reason. I don't think there is a convention for fork check to be `>= ForkSeq.electra` vs `< ForkSeq.electra`. 

Several places use `<` see: https://github.com/ChainSafe/lodestar/blob/94b6c5b0db4e60c7406309367f5b2d7253e8d3c5/packages/state-transition/src/block/initiateValidatorExit.ts#L39 https://github.com/ChainSafe/lodestar/blob/94b6c5b0db4e60c7406309367f5b2d7253e8d3c5/packages/state-transition/src/block/processDeposit.ts#L73 https://github.com/ChainSafe/lodestar/blob/94b6c5b0db4e60c7406309367f5b2d7253e8d3c5/packages/state-transition/src/epoch/processEffectiveBalanceUpdates.ts#L60